scripts/services/getConfig.ts

```diff
@@ -260,14 +260,14 @@ export const getConfig: T.ExpectedExports.getConfig = compat.getConfig({
     "spec": {
       "wt-server": {
         "type": "boolean",
-        "name": "Enable Watchtower Server",
+        "name": "Enable Watchtower",
         "description":
           "Allow other nodes to find your watchtower server on the network.",
         "default": false,
       },
       "wt-client": {
         "type": "boolean",
-        "name": "Enable Watchtower Client",
+        "name": "Enable Watchtower (Client Only)",
```

Comment:
Why name it this way? Does `wt-server` above automatically enable `wt-client`?

Reply:
Yes

Reply:
Maybe be more explicit about this? "Enable watchtower (Server and Client)" seems better than "Enable Watchtower".

Reply:
And again, this strikes me as odd that the user does not have the ability to enable only a server

Reply:
you are absolutely right. I doublechecked, and the server toggle is for server only. I'll adjust the labels and the logic behind my error handling accordingly.

Reply:
done.